src/MoBi.Core/Domain/Model/MoBiSimulation.cs

```diff
@@ -49,6 +49,7 @@ public interface IMoBiSimulation : IWithDiagramFor<IMoBiSimulation>, ISimulation
       IReadOnlyList<IBuildingBlock> BuildingBlocks();
 
       IReadOnlyCollection<OriginalQuantityValue> OriginalQuantityValues { get; }
+      bool HasUntraceableChanges { get; set; }
```

Comment:
I think this guy deserves some comments.
Also it sucks to have a property here that is just required for a one time project conversion...